changelog.d/210.misc

```diff
@@ -0,0 +1 @@
+Ported CI/Cl checks to Github Actions 
```

Comment:
```suggestion
Ported CI checks to Github Actions.
```

I'm not sure if "Cl" (lowercase L) is that common a term, and "CI" should encompass changelog checks as well.

Reply:
Sorted!